content_schemas/formats/shared/definitions/_logo.jsonnet

```diff
@@ -0,0 +1,32 @@
+{
+  type: "object",
+  properties: {
+    formatted_title: {
+      type: "string",
+    },
+    crest: {
```

Comment:
This looks a bit complex with allowing string and null and then having null in the enum. Though this is just a move I wonder if we can simplify it.

I wonder if it'd be better to require a string for this and then to just not set the property all together if this isn't defined (i.e. capitalise on it not being a required property)

Reply:
I've removed both `null` from this property as it's optional anyway.